dataservice/api/diagnosis/schemas.py

```diff
@@ -0,0 +1,36 @@
+from dataservice.api.diagnosis.models import Diagnosis
+from dataservice.api.common.schemas import BaseSchema
+from marshmallow_sqlalchemy import field_for
+from marshmallow import validates, ValidationError
+from dataservice.extensions import ma
+
+MIN_AGE_DAYS = 0
+MAX_AGE_DAYS = 32872
+
+
+class DiagnosisSchema(BaseSchema):
+    # Should not have to do this, since participant_id is part of the
+    # Diagnosis model and should be dumped. However it looks like this is
+    # still a bug in marshmallow_sqlalchemy. The bug is that ma sets
+    # dump_only=True for foreign keys by default. See link below
+    # https://github.com/marshmallow-code/marshmallow-sqlalchemy/issues/20
+    participant_id = field_for(Diagnosis, 'participant_id', required=True,
+                               load_only=True)
+
+    class Meta(BaseSchema.Meta):
+        model = Diagnosis
+
+    @validates('age_at_event_days')
```

Comment:
Wasn't sure what to use for the age range. Looked at kids first dictionary and used the min, max range for age_at_diagnosis_days. See [link](https://github.com/kids-first/kf-dictionary/blob/72aab588f06469ab74abaf406e52ef2d35e50289/gdcdictionary/schemas/diagnosis.yaml#L45)